### PR TITLE
Use Lexis library for ZeroBuf-based vocabulary

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -28,6 +28,7 @@ common_find_package(Boost REQUIRED COMPONENTS filesystem serialization system
                                          unit_test_framework)
 common_find_package(Brion REQUIRED)
 common_find_package(FlatBuffers REQUIRED)
+common_find_package(Lexis)
```
Comment (hernando) [inline CMakeLists.txt]: Not REQUIRED?

Comment (tribal-tec) [inline CMakeLists.txt]: The UI part is optional, and Lexis is only used there right now.

**Comment by hernando** [comment]
I think you have to add:

    pip install pyparsing

to the travis recipe to fix the CI error. I'm trying to reproduce so I'm not fully sure yet.

File: CMakeLists.txt (hunk 1 above)
Comment (hernando) [inline CMakeLists.txt]: I realized that after seeing the whole code review.

File: .gitsubprojects [hunk 2]
```diff
@@ -3,4 +3,5 @@ git_subproject(Servus https://github.com/HBPVIS/Servus 134b674)
 git_subproject(Lunchbox https://github.com/Eyescale/Lunchbox.git b990d7f)
 git_subproject(Brion https://github.com/BlueBrain/Brion.git 4e8898f)
 git_subproject(FlatBuffers https://github.com/BlueBrain/flatbuffers.git 1b70563)
-git_subproject(ZeroEQ https://github.com/HBPVis/ZeroEQ.git d37af92)
+git_subproject(ZeroEQ https://github.com/HBPVis/ZeroEQ.git 1014ea8)
+git_subproject(Lexis https://github.com/HBPVis/Lexis.git c0adc62)
```
Comment (hernando) [inline .gitsubprojects]: Update the SHA1 from Lexis.

Comment (tribal-tec) [inline .gitsubprojects]: In fact it was CMake/common which was outdated.

**Pull request merged**